integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java

```diff
@@ -139,6 +139,9 @@ public synchronized void allocateContainer(Container container) {
     mOutstandingContainerRequestsLatch.countDown();
   }
 
+  /**
+   * Requests for containers.
```

Comment:
The method name already says this. This is a private method so javadoc is not mandatory